src/features/Send/useCases/ListSelectedTokens/EditAmount/DeleteAmountWhenZeroed.ts

```diff
@@ -0,0 +1,26 @@
+import {useNavigation} from '@react-navigation/native'
+
+import {TxHistoryRouteNavigation} from '../../../../../navigation'
+import {useSelectedWallet} from '../../../../../SelectedWallet/Context/SelectedWalletContext'
+import {useTokenInfo} from '../../../../../yoroi-wallets/hooks'
+import {asQuantity, Quantities} from '../../../../../yoroi-wallets/utils/utils'
+import {useSend} from '../../../common/SendContext'
+
+export const useDeleteAmountWhenZeroed = () => {
```

Comment:
only used in 1 component, colocate

Reply:
or move directly in to component

Reply:
this is a use case that why is out :/ one of downsides